cmd/neofs-node/config/cast.go

```diff
@@ -0,0 +1,49 @@
+package config
+
+import (
+	"github.com/spf13/cast"
+)
+
+func panicOnErr(err error) {
+	if err != nil {
+		panic(err)
+	}
+}
+
+// StringSlice reads configuration value
+// from c by name and casts it to []string.
+//
+// Panics if value can not be casted.
+func StringSlice(c *Config, name string) []string {
+	x, err := cast.ToStringSliceE(c.Value(name))
+	panicOnErr(err)
+
+	return x
+}
+
+// StringSliceSafe reads configuration value
+// from c by name and casts it to []string.
+//
+// Returns nil if value can not be casted.
+func StringSliceSafe(c *Config, name string) []string {
+	return cast.ToStringSlice(c.Value(name))
+}
+
+// String reads configuration value
+// from c by name and casts it to string.
+//
+// Panics if value can not be casted.
+func String(c *Config, name string) string {
```

Comment:
I propose to:
- remove unsafe converters until we need one,
- name safe converters without `Safe` prefix.

Reply:
`<Type>` methods will be used for mandatory configurations, and `<Type>Safe` - for optional.

Reply:
Type casting is not connected with requirements, isn't it? If some mandatory field is missing, then `<Type>` function won't panic. Did I get the idea wrong?

```
func TestString(t *testing.T) {
	configtest.ForEachFileType("test/config", func(c *config.Config) {
		require.Panics(t, func() {
			config.String(c, "mandatory_value_that_missing") // no panic there
		})
	})
}
```

Reply:
It will panic if value is not of the particular type (e.g. `String` panics if config can no be interpreted as `string`).